package top.qwq123.baas.schedule.service

import android.app.Service
import android.content.Intent
import android.os.IBinder

class ScheduleExecutionService : Service() {
    override fun onBind(intent: Intent?): IBinder? = null

    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
        // TODO: run scheduled BAAS task
        return START_NOT_STICKY
    }
}
